### ISSUE TITLE
A way to detect busy cores 

### ISSUE THREAD

**Issue by kkrumlian** [OpenClinica/enketo-express-oc#629]
In regards to the headless enketo deployment Is there a way to detect # of active requests/threads/cores at any given time ?

Krikor

**Comment by MartijnR** [comment]
No, not on the Enketo side. I wonder if that is best done on the OS side.
